apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -169,6 +169,16 @@ function load_starter_page_templates() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_starter_page_templates' );
 
+/**
+ * Load Global Styles plugin.
+ */
+function load_global_styles() {
+	if ( is_site_eligible_for_full_site_editing() ) {
```

Comment:
Nice! I think this is a great way of limiting Global Styles to just folks inside our current A/B test as a way of feature gating it without needing theme support. We can always expand this as needed :)

Note for those who don't know: this check will be `true` on standalone/.org sites, so testing the whole FSE plugin in a local environment is still possible.